### DISCUSSION TITLE
Lost/Stuck? WUST

### DISCUSSION THREAD

**Discussion by SNXian** [wormhole-foundation/wormhole#340] [Problems / bugs] [answered]
Hello pls I need help.
I made a transaction of WUST from SOLANA TO ETHEREUM. But got stuck on execution part. Now I dont see my money on Sollet wallet neither on metamask (didnt ask for any interaction with metamask).
Can someone pls help me? Im desperate.
Account : https://explorer.solana.com/address/HmiHJy5VMDAagaHpE6ikWYBWY16kSUVAbfE9JYN1ejqJ
Transaction : https://explorer.solana.com/tx/MyZ1oaEdVofq5ZuSiKWhkgdabGemT1w3J12P9yQjffsJ8Z8cwquqaykL8ZT8WJq2JsZ9s4udDm23gWfHYaLJ6AB

**Comment by leoluk** [comment, marked as answer]
The transaction was successfully processed by the Wormhole network, but the wallet you're using may not have properly sent the Eth transaction. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. If you need help with that, contact the wormholebridge.com team: https://github.com/solana-labs/oyster/issues

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest: c344a3a2bff3895d006f985395c04841e80da7e5ca8a131451c2f51df04c887c

Signed VAA bytes: 

> 01000000010d003ffe94454ff0189c695b04c5694e265be496956395d02ae9ec39cd62c2d5edf65030c771dfc69265f1e679450f970c78b70be1dba149ee3eb1369f86b91ce0b901014afde8beb6fc04bbc0de2def80e9b889e1e85b6d1461d140cde6bea633c67572374426c7e6e0087a0d7bb6b4897afdbd7e2bb5022a99ce41fd58cd21fff575c60102f117e7cad6e27cda8f78d3e4f5009096967fa3409b753a30b8fd9f63ad3e66c8483ba6857f6acc96a1e7e82f4bab8cf1a5d1561d5bb6b388bbccfbdfba9872bf0103a27253c93790aa5550346d39b0e9e5ec42ff0a1cb075639ee9dd0239387a7bad71e5d03a2de37342d614ee508e45e3bb6b2f0a67324a342ecb9fa927aa692e340004a3252f8c46bcb6c70cdaba54003136f2a540acaa3eb68ad87c4acb428f97b92b16440752b500572d5d9db4aa0926ea1557abff73f5b4f9b3a20d0b9160437f15010566ec950811e8d6c28f1ea93237e20057de7adbbda5ab8be21975012d4cf8db4e48f933e0706f1393ccda927fc66a82fd8546eb842c59d2f287fbc237261bbbeb0006d7eaec8860ad5df908ca3b3d7395a5c818451f17ffdc21cb341610a96b96b16b6e417e0d617df812c51f3d0efb71192dc88f763c5cf433bd5d0c6a44078389c900074028176a806e1dadacad3dda7d6a9a29cf847a262c57b4054c3c8e86ea9a5e30105beafd57e9b33159af93a26342c2ed9fd9263e602d3aa9d34d389fc8b8f97b010a8133434c48b3d00e142ee766358584a5e704947dd8dce4b295b1a45b88870f0f6b83f1e4965ee19ad7db9fa02d8f8e2adeee192aad8b68f864eee539c39b752f010b6c8f7eea1919689aa5d818b4dc524a9e55f5f1353277d10846c4997bfc84f7077b487fe6fa712bc0a66f7d7707b177d9b8646095d61e08606189711195d4c75e000d2aeb6bfa4ec5a3f0773a6ac0e32c289cff680b4b915fdff6b908e0f1cf7488763a3f412ef81c862335709f2bca838753c080326a3203d22c6180be457029a27900104f1a55aafc2f1a62d22258fe7418e543ab750919bd45a1f82e93792e153fc5385e69fc8ef000ffdeb24b9853ba5e7183f62c3c0b84616ba838378472d2c4d65101123c70a2aede740bfa53dda83ca2d4913c3d3bc6f1184007674cf12e5f41fe88dc5e792943e3f4e853250de7228830d21fc1aedbf0853628bec90bac30511b0c5f006123044810000143a4010245ba61d4ca1fe80cfa86c7e69558b787db457f042d710a59cd04ebeb51e0570100000000000000000000000003dc54da70469dc3d53d4af0ae43236b6ff4f2b702000000000000000000000000a47c8bf37f92abed4a126bda807a7b7498661acd0900000000000000000000000000000000000000000000000000001ef2c2da3800
